proxy.go

```diff
@@ -52,30 +52,28 @@ func (rp RelengapiProxy) runForever() {
 	// httputil's ReverseProxy is not specifically "reverse", and it will
 	// do fine here.  The director transforms outgoing requests.
 	director := func(req *http.Request) {
-        // redirect tooltool traffic to tooltool.mozilla-releng.net
-        if strings.HasPrefix(req.URL.Path, "/tooltool") {
-            req.URL.Scheme = "https"
-            req.URL.Path = strings.TrimPrefix(req.URL.Path, "/tooltool")
-            req.URL.RawPath = ""
-            req.URL.Host = "tooltool.mozilla-releng.net"
-            req.Host = "tooltool.mozilla-releng.net"
-        // redirect treestatus traffic to treestatus.mozilla-releng.net
-        } else if strings.HasPrefix(req.URL.Path, "/treestatus") {
-            req.URL.Scheme = "https"
-            req.URL.Path = strings.TrimPrefix(req.URL.Path, "/treestatus")
-            req.URL.RawPath = ""
-            req.URL.Host = "treestatus.mozilla-releng.net"
-            req.Host = "treestatus.mozilla-releng.net"
-        // other services then tooltool and treestatus/tooltool are still using
-        // old url. once all services are migrated to new url then we will clean
-        // up this and make this function more generic
-        // https://bugzilla.mozilla.org/show_bug.cgi?id=1393648
-        } else {
-            // point toward the upstream server
-            req.URL.Scheme = "https"
-            req.URL.Host = rp.relengapiHost
-            req.Host = rp.relengapiHost
-        }
+		if strings.HasPrefix(req.URL.Path, "/tooltool") {
+			req.URL.Scheme = "https"
+			req.URL.Path = strings.TrimPrefix(req.URL.Path, "/tooltool")
+			req.URL.RawPath = ""
+			req.URL.Host = "tooltool.mozilla-releng.net"
+			req.Host = "tooltool.mozilla-releng.net"
+		} else if strings.HasPrefix(req.URL.Path, "/treestatus") {
+			req.URL.Scheme = "https"
+			req.URL.Path = strings.TrimPrefix(req.URL.Path, "/treestatus")
+			req.URL.RawPath = ""
+			req.URL.Host = "treestatus.mozilla-releng.net"
+			req.Host = "treestatus.mozilla-releng.net"
+		} else if strings.HasPrefix(req.URL.Path, "/mapper") {
+			req.URL.Scheme = "https"
+			req.URL.Path = strings.TrimPrefix(req.URL.Path, "/mapper")
+			req.URL.RawPath = ""
+			req.URL.Host = "mapper.mozilla-releng.net"
+			req.Host = "mapper.mozilla-releng.net"
+		} else {
+			log.Fatal("Non exising service was requested.")
```

Comment:
1) If a request is made with an invalid path, the server should return status code 404, and not shut down.
2) The log message should include the path that was requested.